README.md

```diff
@@ -309,6 +317,20 @@ https://medium.com/matic-network/understanding-dagger-453d90480c51
   - One way to prevent DDoS is to check state validation when user submits the transaction (before tx goes to pending pool)
   - Node must handle state check validation part while accepting transactions. If any node decides to skip this implementation, other nodes will ignore all future transactions from particular node.
 
+17. Exit
+
+  - Q. Why Matic doesn't have Exit despite of it is like Plasma?
+  - A. Exit is not necessarily needed for sidechain. It is one of security model to achieve customer asset preservation. In our model, Delegates reselection ceremony and fast chain resuming are the security model for assets.
+
+18. Network layer censorship
+
+  - Q. Ethereum doesn't have network layer censorship-resistance for now, but bitcoin have. The reason why Ethereum doesn't have: Using UDP on network layer and cannot use Tor for hidden-mode. If geth apply libp2p, it is possible to be censorship-resistant. Then, how about Matic? Matic is using geth.
+  - A. Bitcoin and Ethereum have been regarded as commodity by U.S. SEC. Now it's lower possibility to censor comodity-cryptocurrency for now. In the future, depends on priority, Geth will implement it and we also apply it on Matic.
+
+19. Delegates reselection ceremony and fast chain resuming
+  - Q. Worst case duration of detecting malicious behaviro is matter. How do you guarantee detection? Otherwise, rollback is getting to be long and double spend can be serious.
+  - A. No idea for now...
```

Comment:
We will use dynasty like Casper based on our checkpoints. Stakers will select delegates at the certain interval and in next dynasty delegates will start producing nodes.

Comment -- We have to change node for that.

Reply:
I think this wlll be the critical path to this question.

Ref
https://github.com/maticnetwork/whitepaper/pull/4/files#r195956749